src/layouts/inner/LayoutDataViewList.jsx

```diff
@@ -0,0 +1,152 @@
+import {
+    Box,
+    Container,
+    Pagination,
+    PaginationItem,
+    Paper, Skeleton, Stack,
+    Typography
+} from "@mui/material";
+import {PageHeader} from "@/components/ui/PageHeader.jsx";
+import {Badge} from "@/components/base/Badge.jsx";
+import {EmptyState} from "@/components/ui/EmptyState.jsx";
+import {Link, useLocation} from "react-router-dom";
+
+/**
+ *
+ * @param header {{title:string, actionButton:ReactNode || foo}}
+ * @param dataSummary {{isVisible:boolean }} [foo]
+ * @param filterView {{}} [foo]
+ * @param dataList
+ * @returns {JSX.Element}
+ * @constructor
+ */
+
+export const LayoutDataViewList = (
+    {
+        header,
+        dataSummary = {isVisible: false},
+        filterView = {isVisible: false, options: []},
+        dataList,
+    }) => {
+    const isLoading = !dataList.children;
+
+    const location = useLocation();
+    const query = new URLSearchParams(location.search);
+    const page = parseInt(query.get('page') || '1', 10);
+
+    return (
+        <Container>
+            <PageHeader title={header.title} actionButton={header.actionButton} isLoading={isLoading}/>
+
+            <Box id="optional-section" sx={{marginY: 8}}>
+                {filterView.isVisible &&
+                    (isLoading ?
```

Comment:
those ternary chain are some complex, would be nice if is possible to extract some logic to outside the render and try to not if>if